api/controllers/mainController.js

```diff
@@ -0,0 +1,41 @@
+const { mainService } = require("../services");
+const { catchAsync } = require("../utils/error");
+
+const getCart = catchAsync(async (req, res) => {
+  const userId = req.user.id;
+
+  const cart = await mainService.getCart(userId);
```

Comment:
```suggestion
  const carts = await cartService.getCarts(userId);
```
userId에 일치하는 카트 개수는 복수가 될 수 있기 때문에 전반적인 네이밍을 carts로 수정해주시는게 좋을 것 같습니다.